aio_kavenegar/client.py

```diff
@@ -0,0 +1,182 @@
+import json
+
+from typing import Literal
+
+import httpx
+
+from aio_kavenegar.exceptions import APIException, HTTPException
+from aio_kavenegar.types import KavenegarResponse
+
+
+# Default requests timeout in seconds.
+DEFAULT_TIMEOUT: int = 10
+DEFAULT_HOST: str = "api.kavenegar.com"
+DEFAULT_HEADERS: dict = {
+    "Accept": "application/json",
+    "Content-Type": "application/x-www-form-urlencoded",
+    "charset": "utf-8",
+}
+
+
+class AIOKavenegarAPI:
+    """
+    https://kavenegar.com/rest.html
+    """
+
+    def __init__(
+        self,
+        apikey: str,
+        timeout: int = DEFAULT_TIMEOUT,
+        host: str = DEFAULT_HOST,
+        headers: dict = DEFAULT_HEADERS,
```

Comment:
Mutable default values [are not recommended](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil), I would use `None` instead.

Reply:
@Alirezaja1384 , I think we can remove them from __all__ to make them unmutable by user and look that as constants

Reply:
Currently default timeout is set like this.